src/index.html

```diff
@@ -11,7 +11,180 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <fieldset>
+        <legend>Personal information</legend>
+      
+        <label class="top">
+            Surname: 
+          <input 
+              type="text"
+              name="Surname"
+              minlength="3"
+              maxlength="15"
+              id="Surname"
+              required
+          >
+        </label>
```

Comment:
style guide is using 2 space instead 4
```suggestion
        <label class="top">
          Surname: 
          <input 
            type="text"
            name="Surname"
            minlength="3"
            maxlength="15"
            id="Surname"
            required
          >
        </label>
```